sympy/functions/elementary/complexes.py

```diff
@@ -775,6 +910,30 @@ def _eval_transpose(self):
 class adjoint(Function):
     """
     Conjugate transpose or Hermite conjugation.
+
+    Examples
+    ========
+
+    >>> from sympy import adjoint
+    >>> from sympy.matrices import MatrixSymbol
+    >>> A = MatrixSymbol('A', 10, 5)
+    >>> adjoint(A)
+    Adjoint(A)
+
+    Parameters
+    ==========
+
+    arg : MatrixSymbol
+        Symbolic representation of a Matrix Object.
+
+    Returns
+    =======
+
+    value : MatrixExpr
+        MatrixExpr is the superclass for matric expressions.
```

Comment:
This doesn't explain what the return value is.

Reply:
Added a one line explanation.

Reply:
I think this adjoint can also work on Matrix objects, in which case the result wouldn't be a matrix expression. Probably better to just say `Matrix` for this case. 

Regardless, it doesn't need to explain what `MatrixExpr` is here.

Reply:
Actually, I think that this class was designed to work with more general non-commutative expressions than Matrix and I see some usage of the same operator in quantum module
We may also have to explain what Adjoint is different from adjoint, although I pretty much expect this is from some bad designs

Reply:
Yes, I think you're right. That explains why it's in this module. We may want to keep this docstring simple for now, and open an issue for figuring out the two things. I've always been a little confused on why these "operator" functions work on plain symbols. I can understand treating `Symbol('x', commutative=False)` as an operator, but even for a default `Symbol('x')` (which is supposed to represent a complex number), both `adjoint(x)` and `Adjoint(x).doit()` don't evaluate.